### DISCUSSION TITLE
流水线可以获取调用的子流水线所报的错误信息

### DISCUSSION THREAD

**Discussion by ychengwang** [Tencent/bk-ci#6096] [Ideas]
项目的流水线中经常会去调用其他的流水线进行操作，子流水线报错后父流水线也只能知道子流水线执行出错了，不能知道具体的错误，需要手动跳转到子流水线去看才知道是什么错误，看能不能提供什么功能能够在父级流水线就能知道子流水线的错误信息，还要考虑一下多层子流水线调用的情况

**Comment by iccengan** [comment]
这块是设计如此，不做改动哈，主要原因：
1. 大部分的流水线报错并不是通过报错信息就能定位或排查的，仍需通过跳转至子流水线来查看流水线编排以及插件配置等等进行定位。因为插件报错通常会与构建环境、提交的代码等相关。所以即使在插件中展示报错信息，大部分时候仍是需要手动跳转到子流水线的。
2. 存在很多如子流水线多个插件报错（同个stage下多个并行job的插件同时报错）、嵌套子流水线等复杂场景。这些信息都放在子流水线插件中呈现会让子流水线插件的输出变得非常复杂。
